src/components/uikit/PickerModal.tsx

```diff
@@ -221,15 +221,15 @@ function ModalContent({
   );
 }
 
-const Wrapper = styled('SafeAreaView', {
+const Wrapper = styled('View', {
   flex: 1,
   justifyContent: 'flex-end',
 });
 
 const Backdrop = Animated.createAnimatedComponent(
   styled('View', {
     absoluteFill: true,
-    backgroundColor: '$backdrop',
+    backgroundColor: '$line1',
```

Comment:
Yaiks, I wouldn't miss use color variables like this 😄 Just define a custom `rgba` color if there is no suitable color in the theme.